src/pages/en/about/security-notice.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Security Notice
+description: If you find a vulnerability in one of our project, please report it using the procedure detailed here
```

Comment:
Two suggestions: One fixes a typo, one is an alternative description which I think is more clear. Use either one.

```suggestion
description: If you find a vulnerability in one of our projects, please report it using the procedure detailed here.
```

```suggestion
description: How to report a security vulnerability in one of our projects.
```